### DISCUSSION TITLE
Data Importer enabled but not showing in "add configuration"

### DISCUSSION THREAD

**Discussion by shaneburrows** [pimcore/pimcore#10341] [Q&A / Support] [answered]
Hi all,

I am currently running v10.1.3 pimcore and would like to enable the data importer. I have followed the steps outlined here https://pimcore.com/docs/data-importer/current/Installation.html and I can see that it's enabled in the Bundles & Bricks section as you can see in the first screen shot below. However when I go into Datahub and click on "Add Configuration" the only option available is "GraphQL", which you can see in the second screen shot. I assume I am missing something simple here so I am hopeful someone can point me in the right direction to get this working.

![image](https://user-images.githubusercontent.com/68840475/133635171-0e9a844b-2e19-44dd-8c98-df06ace33fe1.png)

![image](https://user-images.githubusercontent.com/68840475/133635404-f9fbaf6b-9ded-4a6c-9a02-3ab6a2cbccb8.png)

Thank you

**Comment by fashxp** [comment, marked as answer]
The data importer bundle is not enabled. 
so you probably missed `./bin/console pimcore:bundle:enable PimcoreDataImporterBundle` ... or it went wrong somehow.

**Reply by shaneburrows** [reply]
I knew it was something simple, not sure exactly how I missed that. It's working now.

Thanks for the quick response
